tests/tests/parsers/code_22f1.log

```diff
@@ -22,6 +22,8 @@
 2024-10-10T23:31:09.943782 042  I --- 29:091138 32:022222 --:------ 22F1 003 000306  #
 2024-10-10T23:36:03.885268 042  I --- 29:091138 32:022222 --:------ 22F1 003 000406  #
 
+2024-10-14T19:40:40.840473 056 RQ --- 37:117647 32:022222 --:------ 22F1 001 00      # fan state request by CO2
+
```

Comment:
You need to have a dict after the `#` so that the test is done.

Reply:
Try this:
```
cat tests/tests/parsers/code_22f1.log | grep ' 32:022222 ' | python client.py -lf parse
```

Reply:
For example:
2024-10-10T22:26:38.421878 041  I --- 29:091138 32:022222 --:------ 22F1 003 000206  # {'fan_mode': 'low',         '_scheme': 'vasco',  '_mode_idx': '02', '_mode_max': '06'}
```

Reply:
nice shell cmd! done

Reply:
It helps me if you click 'Resolve conversation` when you're done.